Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.mm

```diff
@@ -195,6 +195,7 @@ @implementation FSTSyncEngineTestDriver {
   IndexFreeQueryEngine _queryEngine;
 
   int _snapshotsInSyncEvents;
+  int _waitForPendingWritesEvents;
```

Comment:
Looks like this variable should be initialized to `0` in the `init` function, similar to `_snapshotsInSyncEvents`.

Reply:
Done (but I didn't see the other initialization, so I added that as well)

Reply:
Sorry, I mistakenly thought line 289 was part of the initialization, but it's actually in its own `reset` function. My Objective-C knowledge is poor -- it could be the case that primitives are set to `0` automatically, I'm not sure (it wouldn't be the case in C++). So this initialization could be redundant (though shouldn't hurt).

Reply:
Objective-C defaults all members to zero, so explicitly making that initialization is redundant.

https://developer.apple.com/library/archive/documentation/General/Conceptual/CocoaEncyclopedia/Initialization/Initialization.html

Reply:
Thanks, Gil! Sebastian, sorry for the redundant comment.

Reply:
Removed again.